src/app/zap-templates/templates/app/callback.zapt

```diff
@@ -170,9 +172,14 @@ void emberAf{{asUpperCamelCase label}}ClusterClientTickCallback(chip::EndpointId
 {{#zcl_clusters}}
 {{#zcl_commands}}
 /**
-* @brief {{parent.name}} Cluster {{name}} Command callback
+* @brief {{parent.name}} Cluster {{name}} Command callback (client side)
 */
-bool emberAf{{asUpperCamelCase parent.label}}Cluster{{asUpperCamelCase name}}Callback(chip::app::Command * commandObj{{#zcl_command_arguments}}, {{asUnderlyingZclType type}} {{asSymbol label}}{{/zcl_command_arguments}});
+bool emberAf{{asUpperCamelCase parent.label}}Cluster{{asUpperCamelCase name}}Callback(chip::app::CommandSender * commandObj{{#zcl_command_arguments}}, {{asUnderlyingZclType type}} {{asSymbol label}}{{/zcl_command_arguments}});
+
+/**
+* @brief {{parent.name}} Cluster {{name}} Command callback (server side)
+*/
+bool emberAf{{asUpperCamelCase parent.label}}Cluster{{asUpperCamelCase name}}Callback(chip::app::CommandHandler * commandObj{{#zcl_command_arguments}}, {{asUnderlyingZclType type}} {{asSymbol label}}{{/zcl_command_arguments}});
```

Comment:
This part doesn't make sense to me.  I'd think that client -> server commands would have a server side callback and server -> client commands have a client side callback, but this is defining both callbacks for everything.

Reply:
Yes, added a filter for client and server commands, wondering if there are some more clear methods instead of isStrEqual